perfkitbenchmarker/providers/gcp/gcp_managed_relational_db.py

```diff
@@ -127,49 +126,41 @@ def _Create(self):
 
     _, _, _ = cmd.Issue()
 
-  def _ParseMemory(self, memory):
-    """
-    Args:
-      memory:
-
-    Returns"""
-
   def _ValidateMachineType(self, memory, cpus):
     """ validated machine configurations.
 
     Args:
-      memory: (string).
+      memory: (int) in MiB.
       cpus: (int).
 
     Raises:
       ValueError.
     """
-    decoder = gce_virtual_machine.MemoryDecoder(option='memory')
-
-    memory = decoder.Decode(memory, '', None)
     if cpus not in [1] + range(2, 32, 2):
       raise ValueError(
-          'CPUs and memory values are invalid. See '
-          'https://cloud.google.com/sql/docs/postgres/create-instance'
-          'for restrictions.')
+          'CPUs (%i) much be 1 or an even number in-between 2 and 32, '
+          'inclusive.' % cpus)
 
     if memory % 256 != 0:
       raise ValueError(
           'Total memory (%dMiB) for a custom machine must be a multiple'
-          'of 256MiB.', memory)
+          'of 256MiB.' % memory)
     ratio = memory / 1024.0 / cpus
     if (ratio < CUSTOM_MACHINE_CPU_MEM_RATIO_LOWER_BOUND or
         ratio > CUSTOM_MACHINE_CPU_MEM_RATIO_UPPER_BOUND):
```

Comment:
Add a comment to the place where these constants are defined (i.e., at the beginning of this file) to state:

where these constants come from (link to doc)

and that these only apply to Postgres.